src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.tag;
+
+/**
+ * Association Class for Tag
+ */
+public class Tagging {
+    
+    public Tag tag;
+    public String action;
```

Comment:
I've seen other people use enums though. Not really sure why String is not the best data type because I need to return the action to be output as part of a string, as mentioned in the exercise.